tests/logictest/Dockerfile

```diff
@@ -1,6 +1,7 @@
 From python:3.10.4-alpine3.15
 
 RUN /usr/local/bin/python -m pip install --upgrade pip && pip install --user mysql-connector six PyHamcrest requests environs fire
+RUN pip install https://github.com/youngsofun/clickhouse-sqlalchemy/archive/refs/heads/databend.zip
```

Comment:
Do we have a better choice?

Reply:
> Do we have a better choice?

There are some conflicts between official clickhouse-sqlalchemy with clickhouse handlers. But we don't want to implement a new one from scratch. So the fastest way is to fix the conflicts in our patches. cc @youngsofun  known it better.

Reply:
If ever possible, please contrite back to the upstream. Open an issue to track the progress and comment here so that we can switch to the upstream when we are ready to go.

Reply:
@Xuanwo my patch is just a temp hack only to make logic test work on click house as quickly as possible, we need another way to support session on clickhouse-sqlalchemy.  We will investigate it later https://github.com/datafuselabs/databend/issues/new?assignees=&labels=&template=30_blank_issue.md.